modules/manage/pages/manage-logging/manage-logging.adoc

```diff
@@ -265,6 +265,11 @@ It is output in common log format and contains several important fields such as
 | `projector`
 | Troubleshooting log for the projector process which is responsible for sending appropriate mutations from Data nodes to Index nodes.
 
+| `rebalance`
+| Contains reports on rebalances that have occurred.
```

Comment:
Not all rebalance, currently we store only the past 5 rebalances.

Reply:
Done.